tests/chainer_tests/functions_tests/connection_tests/test_deconvolution_2d.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestDeconvolution2D(unittest.TestCase):
```

Comment:
You can see the typical usage of parameterized tests [here](https://github.com/pfnet/chainer/blob/6788a84caab30bf8860a5c8e5ba5355ba97bc1d4/tests/cupy_tests/test_ndarray.py#L29)